### PR TITLE
fix: change FEE_SCHEDULE_FILE_PART_UPLOADED to not show as an error

### LINKED ISSUE #2501: `FEE_SCHEDULE_FILE_PART_UPLOADED` marked as error
### Description
Similar to Java SDK [issue](https://github.com/hashgraph/hedera-sdk-java/issues/1950).

When submitting a FileUpdate along with the required FileAppends in order to update the `0.0.111` fees file, getting the receipt returns or throws a ReceiptStatusException. The FEE_SCHEDULE_FILE_PART_UPLOADED response is not an error, but is a response to indicate that the file is not complete and requires more FileAppends until a Fees file is fully uploaded.

### Steps to reproduce

Ran this on local node (the issue showed up when updating the file on TESTNET):
1. Create a FileUpdateTransaction for file `0.0.111` with the proper fee payer and signatures. 
2. Execute the transaction: 
`final var transactionResponse = transaction.execute(client);`
3. Get the receipt:
 `receipt = transactionResponse.getReceipt(client);`

It throw a ReceiptStatusException.

### Additional context

_No response_

### Hedera network

other

### Version

v2.24.1

### Operating system

macOS

### PR DESCRIPTION
**Pull request by ivaylonikolov7**
**Description**:
Updates TransactionResponse to not throw ReceiptStatus error and mark the receipt as finished

**Related issue(s)**: #2501

Fixes #2501

**Checklist**
- [x] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

File: src/transaction/TransactionReceiptQuery.js [hunk 1]
```diff
@@ -257,6 +257,8 @@ export default class TransactionReceiptQuery extends Query {
                 return [status, ExecutionState.Retry];
             case Status.Success:
                 return [status, ExecutionState.Finished];
+            case Status.FeeScheduleFilePartUploaded:
```
Comment (0xivanov) [inline src/transaction/TransactionReceiptQuery.js:260]: merge with the above case

**Review by 0xivanov — APPROVED** [review]
LG

**Review by ivaylogarnev-limechain — APPROVED** [review]
LGTM

**Review by agadzhalov — APPROVED** [review]
LGTM

**Pull request merged**